frontend/javascripts/admin/project/project_list_view.js

```diff
@@ -85,11 +89,18 @@ class ProjectListView extends React.PureComponent<PropsWithRouter, State> {
   }
 
   async fetchData(): Promise<void> {
-    const projects = await getProjectsWithOpenAssignments();
-
+    let projects;
+    let taskTypeName = null;
+    if (this.props.taskTypeId) {
+      projects = await getProjectsForTaskType(this.props.taskTypeId);
+      taskTypeName = await getTaskType(this.props.taskTypeId);
```

Comment:
You can combine these two lines via `Promise.all` so that only one `await` is used. As a result, both requests are processed in parallel instead of sequentially :)

Reply:
That's cool! Thanks :)